engines/hugo/route.cpp

```diff
@@ -75,36 +75,20 @@ void Route::setDirection(const uint16 keyCode) {
 
 	// Set first image in sequence
 	switch (keyCode) {
-	case Common::KEYCODE_UP:
-	case Common::KEYCODE_KP8:
+	case kActionMoveTop:
```

Comment:
For this function, can keyCode be renamed to avoid confusion? Although it is an action, I'm inclined to suggest direction to match Route::setWalk